types/koa-webpack/koa-webpack-tests.ts

```diff
@@ -0,0 +1,47 @@
+import * as Koa from 'koa';
+import * as webpack from 'webpack';
+import * as koaWebpack from 'koa-webpack';
```

Comment:
Since this is not an ES6 module, you'll want to use CommonJS conventions.

```ts
import koaWebpack = require('koa-webpack');
```

You can then also delete the `declare namespace koaWebpack` line from `index.d.ts` since what it is, is actually a hack to let people use ES6 syntax for things that aren't ES6 modules.